README.md

```diff
@@ -1,22 +1,57 @@
-  <img src="logos/All%20Logos/BookBrainz.svg" alt="BookBrainz" title="BookBrainz" width="198">
-  <img src="logos/All%20Logos/AccousticBrainz.svg" alt="AcousticBrainz" title="AcousticBrainz" width="198">
-  <img src="logos/All%20Logos/CritiqueBrainz.svg" alt="CritiqueBrainz" title="CritiqueBrainz" width="198">
-  <img src="logos/All%20Logos/MusicBrainz%20Picard.svg" alt="Picard" title="Picard" width="198">
-  <img src="logos/All%20Logos/CoverArtArchive.svg" alt="CoverArtArchive" title="CoverArtArchive" width="198">
-  <img src="logos/All%20Logos/MessyBrainz.svg" alt="MessyBrainz" title="MessyBrainz" width="198">
-  <img src="logos/All%20Logos/MetaBrainz%20Community.svg" alt="MetaBrainz Community" title="MetaBrainz Community" width="198">
-</div>
+# Guidelines
+
+## Sizes
+Use logo sizings consistently across projects. For instance, *Brainz logo mini* is used as the top left logo across all MB database project pages. When unsure, refer to prior projects and useage.</br>
+Although the icons should never be smaller than the minimum sizes\*, you may increase sizes as you wish. For instance, the usually small *Brainz icon* can be used in a very large format, if required.
+
+><img src="logos/MetaBrainz/SVG/MetaBrainz_logo_icon.svg" alt="MetaBrainz icon" height="24px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_mini.svg" alt="MetaBrainz logo mini" height="23px"></br>
+>*Brainz icon / Brainz logo mini*</br>
+>No minimum
+
+><img src="icons/metabrainz-icon-detail.svg" alt="MetaBrainz icon detail" height="34px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_small.svg" alt="MetaBrainz logo small" height="33px"></br>
+>*Brainz icon detail / Brainz logo small*</br>
+>Minimum height: 33px/12mm</br>
+>Under this size details can get unclear and distracting, particularly for visually impaired users.
+
+><img src="logos/MetaBrainz/SVG/MetaBrainz_logo_no_text.svg" alt="MetaBrainz logo no text" height="58px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo.svg" alt="MetaBrainz logo" height="55px"></br>
+>*Brainz logo no text / Brainz logo*</br>
+>Minimum height: 55px/20mm</br>
+>Ideally has generous sizing and space to breathe. If the page/app/context is very busy, consider using a simpler icon instead.
+
+\*a logo may display fine at a smaller size, but other project logos have more detail, and won’t. Smaller sizes makes consistent logo display across projects difficult and is to be avoided. However there may be fringe cases it is appropriate.
+
+## Spacing
+><img src="documentation/images/logo_margin.png" alt="stretched logo" height="137px"></br>
+>Allow for enough space around Brainz project logos. Ideally, one ‘half honeycomb’ on all sides.
+
+## Don'ts
+
+><img src="documentation/images/dont_stretch.png" alt="stretched logo" height="33px"></br>
+>*Don't* stretch the logo.
+
+><img src="documentation/images/dont_effects.png" alt="logo with drop shadow effect added" height="33px"></br>
+>*Don't* alter the logo
```

Comment:
> The ListenBrainz logo has changed since the redesign, and all of the LB files (illustrator as well as exports) need to be updated to reflect that.
> Here is the illustrator file for v2 of the logo: https://ufile.io/eccjm5zg
> We should probably keep the old version too for posterity (at least the illustrator files?).

Thanks Monkey!

While updating the LB files I've come across an inconsistency I didn't spot before, in regards to using Roboto or Roboto Condensed. In the 'factory' files it's usually Condensed, but based on what we've actually used on the site icons etc it should probably just be Roboto? Or did you want Condensed for some of the narrower formats.

(this will cause most of the 'factory' exported files to get updated again)